db/src/androidTest/java/io/lantern/db/DBTest.kt

```diff
@@ -534,118 +516,47 @@ class DBTest {
     }
 
     @Test
-    fun testTail() {
```

Comment:
I took out tail because it turned out to be somewhat inefficient and I don't use it.